syncopy/tests/test_connectivity.py

```diff
@@ -648,25 +693,27 @@ def test_corr_cfg(self):
                      cfg=get_defaults(cafunc))
 
     @skip_low_mem
-    def test_corr_parallel(self, testcluster=None):
-
-        ppl.ioff()
-        client = dd.Client(testcluster)
-        all_tests = [attr for attr in self.__dir__()
-                     if (inspect.ismethod(getattr(self, attr)) and 'parallel' not in attr)]
-
-        for test in all_tests:
-            test_method = getattr(self, test)
-            test_method()
-        client.close()
-        ppl.ion()
+    def test_parallel(self):
+        check_parallel(TestCorrelation())
 
     def test_corr_polyremoval(self):
 
         call = lambda polyremoval: self.test_corr_solution(polyremoval=polyremoval)
         helpers.run_polyremoval_test(call)
 
 
+def check_parallel(TestClass, testcluster=None):
```

Comment:
This makes a lot of sense :+1: You could even move this function to `/tests/helpers.py` ;)